### DISCUSSION TITLE
Wheel/Scroll Event Not Working

### DISCUSSION THREAD

**Discussion by zackdove** [marcofugaro/threejs-modern-app#16] [Q&A]
Does Three-Modern-App capture wheel events?

I dont have any wheel events or e.preventDefault calls in my app, but I cant seem to scroll away from the canvas

**Comment by marcofugaro** [comment]
Are you using the orbit controls? They don't let you scroll because the wheel is used for zooming.

See https://github.com/mrdoob/three.js/blob/9d332c1c726e6058fa484c88b9bad3d3e36b9250/examples/jsm/controls/OrbitControls.js#L1250
